devtools/win-install-coreclr.cmd

```diff
@@ -0,0 +1,9 @@
+rem from https://docs.asp.net/en/latest/getting-started/installing-on-windows.html#install-asp-net-5-from-the-command-line
+
+rem install .NET version manager (DNVM)
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}"
+
+rem install .NET execution environment (DNX) for coreclr
+dnvm upgrade -r coreclr
+
+pause
```

Comment:
does it not? sure enough, doesn't seem like it - wonder why github doesn't show that online for new files, only for patches that remove the newline at eof. Apparently I only edited this file with notepad2-mod, not atom. That's an easy fix.